ios/FluentUI/People Picker/ContactCollectionViewLayout.swift

```diff
@@ -0,0 +1,151 @@
+    @objc weak var delegate: UICollectionViewDelegateFlowLayout?
+
+    struct Constants {
+        static let itemWidth: CGFloat = 70.0
+        static let minLineSpacing: CGFloat = 14.0
+        static let maxLineSpacing: CGFloat = 16.0
+
+        static let extraSmallContentContactHeight: CGFloat = 115.0
+        static let smallContentContactHeight: CGFloat = 117.0
+        static let mediumContentContactHeight: CGFloat = 118.0
+        static let largeContentContactHeight: CGFloat = 121.0
+        static let extraLargeContentContactHeight: CGFloat = 125.0
+        static let extraExtraLargeContentContactHeight: CGFloat = 129.0
+        static let extraExtraExtraLargeContentContactHeight: CGFloat = 135.0
+    }
+
+    override init() {
+        super.init()
+        delegate = self
+        minimumLineSpacing = Constants.maxLineSpacing
+    }
+
+    required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) not implemented")
+    }
+
+    override var collectionViewContentSize: CGSize {
+        guard let collectionView = collectionView else {
+            return .zero
+        }
+
+        // TODO: Might not even need the setting here since the delegate method already sets it
+        if let minLineSpacing = delegate?.collectionView?(collectionView, layout: self, minimumLineSpacingForSectionAt: 0) {
+            minimumLineSpacing = minLineSpacing
+        }
+
+        let indexPath = IndexPath(item: 0, section: 0)
+        let numItems = collectionView.numberOfItems(inSection: 0)
+        let height = delegate?.collectionView?(collectionView, layout: self, sizeForItemAt: indexPath).height
+        let totalWidth = numItems > 0 ? CGFloat(numItems) * Constants.itemWidth + (CGFloat(numItems) - 1) * minimumLineSpacing : 0.0
+
+        let size = CGSize(width: totalWidth, height: height!)
+
+        return size
+    }
+
+    override func layoutAttributesForElements(in rect: CGRect) -> [UICollectionViewLayoutAttributes]? {
+        var layoutAttributes = [UICollectionViewLayoutAttributes]()
+
+        if let collectionView = collectionView {
+            numberOfItems = collectionView.numberOfItems(inSection: 0)
+        }
+
+        for itemIndex in 0..<numberOfItems {
+            let indexPath = IndexPath(item: itemIndex, section: 0)
+            if let attributes = layoutAttributesForItem(at: indexPath) {
+                if attributes.frame.intersects(rect) {
+                    layoutAttributes.append(attributes)
+                }
+            }
+        }
+
+        return layoutAttributes
+    }
+
+    override func layoutAttributesForItem(at indexPath: IndexPath) -> UICollectionViewLayoutAttributes? {
+        let attributes = UICollectionViewLayoutAttributes(forCellWith: indexPath)
+        let frame = CGRect(x: CGFloat(indexPath.item) * Constants.itemWidth + CGFloat(indexPath.item) * minimumLineSpacing, y: 0, width: Constants.itemWidth, height: itemSize.height)
```

Comment:
nit: let's use multiple lines for this for readability.
example:
let frame = CGRect(x: CGFloat(indexPath.item) * Constants.itemWidth + CGFloat(indexPath.item) * minimumLineSpacing, 
y: 0,
...

Reply:
Great suggestion! I'll make the required changes.